spark/src/main/java/org/apache/iceberg/actions/Actions.java

```diff
@@ -88,6 +89,10 @@ public ExpireSnapshotsAction expireSnapshots() {
     return new ExpireSnapshotsAction(delegate);
   }
 
+  public BaseDropTableSparkAction dropTableAction() {
```

Comment:
Let's not add it to the old Actions API which we are about to deprecate.

Reply:
I think I know why this adds it here. Do don't have an implementation of `ActionsProvider` yet. I am about to add it so we will move this logic. It is ok to keep it here for now.